transport/rif_comms/client.py

```diff
@@ -72,13 +71,14 @@ def is_subscribed_to(self, rsk_address: Address) -> bool:
             )
         ).value
 
-    def send_message(self, payload: str, topic_id: str):
+    def send_message(self, payload: str, rsk_address: Address):
         """
         Sends a message to a destination RSK address.
         Invokes the SendMessageToTopic GRPC API endpoint.
         :param payload: the message data to be sent
-        :param topic_id: the topic identifier to be sent to
+        :param rsk_address: the destination for the message to be sent to
         """
+        topic_id = self._get_peer_id(to_checksum_address(rsk_address))
```

Comment:
stuff like this should be looked at [here](https://jirainfuy.atlassian.net/browse/RLP-924)